.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path')
+
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: [
+      path.resolve(__dirname, './packages/**/tsconfig.eslint.json'),
```

Comment:
Would be happy if this did not need to move out of `package.json`, but I see you are using `path`. I won't have strong feelings about it if there is no workaround. :thinking:

Reply:
I moved it inside `pacakge.json` again! I thought missing `path` and `__dirname` was the problem but it turned out weren't